docs/current_docs/manuals/administrator/engine/472258-custom-runner.mdx

```diff
@@ -0,0 +1,81 @@
+---
+slug: /manuals/administrator/472258/custom-runner
+---
+
+# Use a Custom Runner
+
+A runner is the "backend" of the Dagger Engine where containers are actually executed.
+
+Runners are responsible for:
+
+- Executing containers specified by functions
+- Pulling container images, Git repos and other sources needed for function execution
+- Pushing container images to registries
+- Managing the cache backing function execution
+
+The runner is distributed as a container image, making it easy to run on various container runtimes like Docker, Kubernetes, Podman, etc.
+
+The consolidated steps to use a custom runner are:
+
+- Determine the runner version required by checking the release notes of the CLI or SDK you intend to use.
+- If changes to the base image are needed, make those and push them to a registry. If no changes are needed, just use it as is.
+- Start the runner image in your target of choice, [requirements](#execution-requirements) and [configuration](#configuration) in mind.
+- Export the `_EXPERIMENTAL_DAGGER_RUNNER_HOST` environment variable with a [a value pointing to your target](#connection-interface).
+- Call `dagger call` or execute SDK code directly with that environment variable set.
+
+:::note
+The `_EXPERIMENTAL_DAGGER_RUNNER_HOST` variable is experimental and may change in future.
+:::
+
+## Distribution and Versioning
+
+The runner is distributed as a container image at `registry.dagger.io/engine`.
+
+- Tags are made for the version of each release.
+- For example, the [`v0.3.7` release](https://github.com/dagger/dagger/releases/tag/v0.3.7) has a corresponding image at `registry.dagger.io/engine:v0.3.7`
```

Comment:
I would update this to a newer tag, e.g. `v0.10.2`

Reply:
Done